<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YouTubeDlGui</class>
 <widget class="QMainWindow" name="YouTubeDlGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YouTube DL GUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/resources.qrc">
    <normaloff>:/images/youtube.png</normaloff>:/images/youtube.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnPromote">
        <property name="text">
         <string>&amp;Up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDemote">
        <property name="text">
         <string>&amp;Down</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemove">
        <property name="text">
         <string>&amp;Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnDownload">
        <property name="text">
         <string>Down&amp;load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClearFinished">
        <property name="text">
         <string>Clear Finished</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLineEdit" name="txtUrl">
      <property name="placeholderText">
       <string>YouTube Video URL</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="cmbDownloadOptions"/>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="DownloadQueue" name="tblDownloadQueue">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnCollapse">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/resources.qrc">
          <normaloff>:/images/expand.png</normaloff>:/images/expand.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>10</width>
          <height>6</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblAdvanced">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Show Advanced</string>
        </property>
        <property name="buddy">
         <cstring>btnCollapse</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btnAdd">
      <property name="text">
       <string>&amp;Add</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QFrame" name="frameAdvanced">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblVideoFormat">
           <property name="text">
            <string>Video Format</string>
           </property>
           <property name="buddy">
            <cstring>cmbVideoFormat</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbVideoFormat"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="lblAudioFormat">
           <property name="text">
            <string>Audio Format</string>
           </property>
           <property name="buddy">
            <cstring>cmbAudioFormat</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbAudioFormat"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/images/quit.png</normaloff>:/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/images/preferences.png</normaloff>:/images/preferences.png</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DownloadQueue</class>
   <extends>QTableWidget</extends>
   <header>ui/downloadqueue.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtUrl</tabstop>
  <tabstop>cmbDownloadOptions</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnCollapse</tabstop>
  <tabstop>cmbVideoFormat</tabstop>
  <tabstop>cmbAudioFormat</tabstop>
  <tabstop>btnPromote</tabstop>
  <tabstop>btnDemote</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>btnDownload</tabstop>
  <tabstop>tblDownloadQueue</tabstop>
 </tabstops>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
